#
# $Id: httpd.conf-2.0.patch,v 4.3 2005/04/29 22:38:31 matt Exp $
#

# Author: Matt Simerson
#
# httpd.conf diffs for Apache 2.0 for Mail::Toaster
# apply this to the stock httpd.conf.
#


*** httpd.conf.orig     Wed Nov 24 19:14:30 2004
--- httpd.conf  Sat Nov 27 16:44:01 2004
***************
*** 386,392 ****
  # http://httpd.apache.org/docs-2.0/mod/core.html#options
  # for more information.
  #
!     Options Indexes FollowSymLinks
  
  #
  # AllowOverride controls what directives may be placed in .htaccess files.
--- 386,392 ----
  # http://httpd.apache.org/docs-2.0/mod/core.html#options
  # for more information.
  #
!     Options Indexes FollowSymLinks ExecCGI
  
  #
  # AllowOverride controls what directives may be placed in .htaccess files.
***************
*** 403,408 ****
--- 403,422 ----
  
  </Directory>
  
+ <Directory "/usr/local/www/squirrelmail">
+     Options Indexes ExecCGI
+     AllowOverride None
+     Order allow,deny
+     Allow from all
+ </Directory>
+ 
+ <Directory "/usr/local/share/isoqlog/htmltemp/images">
+     Options None
+     AllowOverride None
+     Order allow,deny
+     Allow from all
+ </Directory>
+ 
  #
  # UserDir: The name of the directory that is appended onto a user's home
  # directory if a ~user request is received.
***************
*** 439,445 ****
  # negotiated documents.  The MultiViews Option can be used for the 
  # same purpose, but it is much slower.
  #
! DirectoryIndex index.html index.html.var
  
  #
  # AccessFileName: The name of the file to look for in each directory
--- 453,459 ----
  # negotiated documents.  The MultiViews Option can be used for the 
  # same purpose, but it is much slower.
  #
! DirectoryIndex index.cgi index.shtml index.html index.htm index.html.var
  
  #
  # AccessFileName: The name of the file to look for in each directory
***************
*** 595,600 ****
--- 609,617 ----
  # We include the /icons/ alias for FancyIndexed directory listings.  If you
  # do not use FancyIndexing, you may comment this out.
  #
+ Alias /squirrelmail/ "/usr/local/www/squirrelmail/"
+ Alias /isoqlog/images/ "/usr/local/share/isoqlog/htmltemp/images/"
+ Alias /phpMyAdmin/ "/usr/local/www/phpMyAdmin/"
  Alias /icons/ "/usr/local/www/icons/"
  
  <Directory "/usr/local/www/icons">
***************
*** 903,909 ****
  # To use CGI scripts outside of ScriptAliased directories:
  # (You will also need to add "ExecCGI" to the "Options" directive.)
  #
! #AddHandler cgi-script .cgi
  
  #
  # For files that include their own HTTP headers:
--- 920,928 ----
  # To use CGI scripts outside of ScriptAliased directories:
  # (You will also need to add "ExecCGI" to the "Options" directive.)
  #
! AddHandler cgi-script .cgi
! AddType application/x-httpd-php .php
! AddType application/x-httpd-php-source .phps
  
  #
  # For files that include their own HTTP headers:
